R/summarise_draws.R

```diff
@@ -60,14 +64,66 @@ summarize_draws <- summarise_draws
 
 
 #' @export
-summarise_draws.default <- function(x, ...) {
-  x <- as_draws(x)
-  summarise_draws(x, ...)
+summarise_draws.default <- function(.x, ...) {
+  .x <- as_draws(.x)
+  summarise_draws(.x, ...)
+}
+
+# Helper functions
```

Comment:
Can you move all internal non-exported functions below the exported functions to make it easier to read through the file (someone only needs to look at the internal functions if they really want to understand the guts)